.github/workflows/build.yml

```diff
@@ -36,3 +36,32 @@ jobs:
       - name: Test import
         run: |
           python -c "import compas_ags"
+
+  build-docs:
+    needs: build-packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install cython --install-option='--no-cython-compile'
+      - name: Install
+        run: |
+          pip install --no-cache-dir -r requirements-dev.txt
+      - name: Build docs with sphinx
+        run: |
+          rm -rf dist/docs && invoke docs
+      - name: Deploy docs
+        if: success()
+        uses: crazy-max/ghaction-github-pages@v2
+        with:
+          target_branch: gh-pages
+          build_dir: dist/docs
+          keep_history: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
can you remove this? you merged incorrectly with master and reintroduced this...

Reply:
ok

Reply:
now this is equal the master